conf/questions.yml

```diff
@@ -26,15 +26,15 @@ questions:
   -
     slug: 'question_mailinglist'
     title: 'Do you have a mailing list?'
-    answer: 'Yes. Subscribe to our Google Group [here](https://groups.google.com/forum/#!forum/grails-dev-discuss).'
+    answer: 'No, we prefer to converse over GitHub, Slack, or Stack Overflow.'
   -
     slug: 'question_feature'
     title: 'Where can I suggest a new feature or share an interesting finding?'
-    answer: 'The [Grails mailing list](https://groups.google.com/forum/#!forum/grails-dev-discuss) is a great place to start a conversation with the Grails developers and other Grails users.'
+    answer: 'The [Grails Community Slack](https://slack.grails.org) is a great place to start a conversation with the Grails developers and other Grails users.'
```

Comment:
> The [Grails Community Slack](https://slack.grails.org) is a great place to start a conversation with the Grails developers and other Grails users.

This is a response to "Where can I suggest a new feature or share an interesting finding?".  I don't think Stackoverflow is a good place for that.  I would suggest https://github.com/orgs/grails/discussions.